core/sqf/src/seatrans/hbase-trx/src/main/java/org/apache/hadoop/hbase/coprocessor/transactional/SplitBalanceHelper.java

```diff
@@ -304,11 +304,13 @@ protected void zkCleanup() {
           List<String> trafTables = ZKUtil.listChildrenNoWatch(zkw, zSplitBalPathNoSlash);
           List<String> hbaseTables = ZKUtil.listChildrenNoWatch(zkw, SplitBalanceHelper.zkTable);
           
-          for(String tableName : trafTables) {
+          if(trafTables != null && hbaseTables != null) {

```

Comment:
you are right Selva, thanks to help me understand this code.
It now looks good for me.